### PR TITLE
[238] v0.5.0 and update changelog

### PR DESCRIPTION
**Pull request by trel**
please eyeball the release notes.

otherwise, seems ready to go.

```
$ twine check dist/*
Checking dist/irods_capability_automated_ingest-0.5.0-py3-none-any.whl: PASSED
Checking dist/irods-capability-automated-ingest-0.5.0.tar.gz: PASSED
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,53 @@
...
+- Bump werkzeug from 2.2.3 to 2.3.8              
+- Bump redis from 2.10.6 to 4.4.4                
+- [#215] Fix test failures                       
+- [#220] Update test environment                 
+- [#180] Update supported Python versions        
+- [#180] black formatter - no functional changes 
+- [#212] changed REGISTER to REGISTER_SYNC       
+- [#207] multi read and write from S3 to iRODS for put, putsync
+- [#129] Added functionality for PUT, PUT_SYNC with S3 via Minio
+- [#129] put_sync functionality for data in S3
```
Comment (alanking) [inline CHANGELOG.md:49]: There's some weird trailing space on most of these lines. Feel free to ignore

**Review by alanking — APPROVED** [review]
I saw one thing that may be a non-issue

File: CHANGELOG.md (hunk 1 above)
Comment (trel) [reply CHANGELOG.md:49]: ah, good - will clean

Comment (trel) [reply CHANGELOG.md]: resolving

**Comment by trel** [comment]
tweaked, please eyeball once more

**Review by alanking — APPROVED** [review]
Yes, good. Please squash and #

**Review by korydraughn — COMMENTED** [review]
There's been several updates to dependencies leading up to this release. Is the requirements.txt file in sync with those changes?

**Comment by korydraughn** [comment]
I'm guessing it is if they were all handled by dependabot.

**Comment by alanking** [comment]
Right, updating the requirements file sort of *is* the bump. See #239 for an example: https://github.com/irods/irods_capability_automated_ingest/pull/239/files

**Review by korydraughn — APPROVED** [review]
I looked over the dependencies changes and everything looks good.

Let's get it in.

**Comment by trel** [comment]
a fine question - and yes, i think we're good.

**Comment by trel** [comment]
squashed and #'d

**Pull request merged**